tests/dummy/app/pods/docs/guide/migration-4-0-to-5-0/template.md

```diff
@@ -34,24 +34,6 @@ Changes:
 * `interval` was removed from the format-relative helper
 * `now` was removed from the format-relative helper
 
-### **Translations**
-
-* Special character escaping in translations is now done via a single quote, `'` instead of the previous slash `\`.
```

Comment:
I need to restore this one sentence.  Escaping still changed to be more ICU spec compliant - it's just no longer necessary for HTML tags and attributes.